diff --git a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -281,3 +281,6 @@ required-features = ["mssql"]
 name = "mssql-macros"
 path = "tests/mssql/macros.rs"
 required-features = ["mssql", "macros"]
+
+[patch.crates-io]
+ahash = { git = "https://github.com/tkaitchuck/ahash.git", tag = "v0.7.6" }
diff --git a/sqlx-core/src/sqlite/statement/handle.rs b/sqlx-core/src/sqlite/statement/handle.rs
--- a/sqlx-core/src/sqlite/statement/handle.rs
+++ b/sqlx-core/src/sqlite/statement/handle.rs
@@ -384,6 +384,7 @@ impl Notify {
         }
     }

+    #[allow(let_underscore_lock)]
     fn wait(&self) {
         let _ = self
             .condvar
